crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -87,91 +83,30 @@ impl ParsedQObject {
         })
     }
 
+    fn parse_impl_method(&mut self, method: &ImplItemMethod) -> Result<()> {
+        // Determine if this method is an invokable
+        if attribute_find_path(&method.attrs, &["qinvokable"]).is_some() {
```

Comment:
we know the index of the `qinvokable` here should we do `if let Some(index) = ...` then pass in the index? Like we do for `ParsedQObject::from_struct(&ItemStruct, attr_index: usize)` ?